src/shared/utils/helpers.ts

```diff
@@ -1,3 +1,30 @@
+import * as z from 'zod';
+
+export const userSchema = z.object({
```

Comment:
okay, so looking at the object, it looks like it's validating a form? but variable name `userSchema` sounds like it's validating a user object?

also, is this validating one specific form or multiple? I would co-locate this with the form if it's just for one form.